corehq/apps/repeaters/models.py

```diff
@@ -315,6 +318,16 @@ def handle_exception(self, exception, repeat_record):
         generator = self.get_payload_generator(self.format_or_default_format())
         return generator.handle_exception(exception, repeat_record)
 
+    @property
+    def is_form_repeater(self):
```

Comment:
i think conventionally we define this on the base class as NotImplemented and then on the FormRepeater class we have it `return True`